### ISSUE TITLE
Default GPU memory usage

### ISSUE THREAD

**Issue by MRCWirtz** [MIC-DKFZ/nnUNet#658]
Hi @FabianIsensee , 

I got a bit confused about the default memory usage of the standard ExperimentPlanner3D_v21: from the supplementary notes (section 9.2) of the nnUNet publication I understood that by default 11 GB RAM is used for training, however, in the comment of the following script it says 7.5 GB:

https://github.com/MIC-DKFZ/nnUNet/blob/e29f7c04f3c5458033ee522ad6145db780d3f510/nnunet/experiment_planning/alternative_experiment_planning/experiment_planner_baseline_3DUNet_v21_16GB.py#L64

Could you clarify which value is correct? Also you might want to change the quoted comment with the 11GB RAM since it is in the 16GB experiment planner script ;-)

Thanks in advance!

Best,
Marcus

**Comment by FabianIsensee** [comment]
HI Marcus, the reason this is this way is historic. nnU-Net was initially designed without mixed precision and for a target GPU memory of 12 GB (TitanX). We later added mixed precision as the default, which brought down GPU memory usage to ~7.5G. At the same time, however, we allowed the inputs to be a tiny bit larger, increasing GPU memory consumption a tiny bit. Since our GPU memory consumption estimate is as handwavy as it gets (and we did not observe benefits of using more GPU memory) we left it as it is. In practice, nnU-Net effectively uses anywhere between 7 and 9GB of GPU memory. We like to give 11GB as requirement so that it is essentially guaraneteed to not crash, even when someone is running it in the same GPU they run their OS GUI on.
Best,
Fabian
